pkg/controllers/user/logging/utils/syslog.go

```diff
@@ -113,3 +113,23 @@ func getSeverity(severityStr string) syslog.Priority {
 
 	return syslog.LOG_INFO
 }
+
+func getWrapSeverity(severity string) string {
+	severityMap := map[string]string{
+		"emerg":   "emerg",
+		"alert":   "alert",
+		"crit":    "crit",
+		"err":     "err",
+		"warning": "warn",
+		"notice":  "notice",
+		"info":    "info",
+		"debug":   "debug",
+	}
+
+	wrapSeverity := severityMap[severity]
+	if wrapSeverity == "" {
+		wrapSeverity = "notice"
```

Comment:
I still don't understand why you return a hardcoded value when mismatch. I would like to return `severity` if the `severityMap` isn't hit.